bin/tools-shared-get-v1.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+get:quarks-utils:bin() {
+  GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+  cd $GIT_ROOT
+
+  # Gets the quarks-utils version from go.mod
+  quarks_utils_version=$(grep code.cloudfoundry.org/quarks-utils go.mod | awk '{print $2}')
+  quarks_utils_version=$(echo $quarks_utils_version | sed 's/.*-//g') # Keep only the commit
+  if [ "$quarks_utils_version" = "utils" ]; then
+    echo "failed to parse commit from go.mod"
+    exit 1
+  fi
+
+  mkdir -p tools/quarks-utils
+  wget -qO - https://github.com/cloudfoundry-incubator/quarks-utils/archive/"$quarks_utils_version".tar.gz | \
+    tar -C tools/quarks-utils -xz --strip-components=1 --no-anchored bin/
+}
+
+get:quarks-utils:bin
```

Comment:
Side note, the naming breaks sh compatibility:

```
~/_git/cf-operator 172598851-docs*
❯ curl https://raw.githubusercontent.com/cloudfoundry-incubator/quarks-utils/e7070658cbfbf1ad34cd679ad4334902e39ac493/bin/tools-shared-get-v1.sh | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   693  100   693    0     0   1188      0 --:--:-- --:--:-- --:--:--  1186
sh: line 18: `get:quarks-utils:bin': not a valid identifier

~/_git/cf-operator 172598851-docs*
❯ curl https://raw.githubusercontent.com/cloudfoundry-incubator/quarks-utils/e7070658cbfbf1ad34cd679ad4334902e39ac493/bin/tools-shared-get-v1.sh | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   693  100   693    0     0   2287      0 --:--:-- --:--:-- --:--:--  2279
```

Reply:
Thanks, apparently my sh is symlinked to bash ...